one-to-one-scheduler/one-to-one-scheduler.gs

```diff
@@ -626,43 +625,161 @@ function sendInvitesToMentors() {
   showInfo(TITLE_SUCCESS, summary);
 }
 
-function cancelEvents() {
-  let calendar = getCalendar();
-  if (!calendar) {
-    showInfo(TITLE_ERROR, MSG_CALENDAR_NOT_FOUND);
+function getRecordingAndChatLinks() {
+  let setupResult = initSetup();
+  if (!setupResult.success) {
     return;
   }
 
-  if (!confirmCalendar(calendar)) {
-    showInfo(TITLE_ERROR, MSG_CALENDAR_REJECTED);
-    return;
+  let noMeetingRows = [];
+  let invalidMeetings = [];
+  let noRecordingLinks = [];
+  let noChatLinks = [];
+  let numSuccess = 0;
+  let numSkipped = 0;
+  let calendarId = setupResult.calendar.getId();
+
+  for (let rowIndex = 0; rowIndex < setupResult.oneToOneData.length; ++rowIndex) {
+    let rowNumber = rowIndex + DATA_BEGIN_ROW;
+    let opsRequestFlag = parseInt(setupResult.activeSheet.getRange(rowNumber, OPS_REQUEST_INDEX + 1).getValue());
+    if (opsRequestFlag !== FLAG_SEND_INVITE) {
+      // The row should be skipped
+      ++numSkipped;
+      continue;
+    }
+
+    let meetIdCell = setupResult.activeSheet.getRange(rowNumber, MEETING_ID_INDEX + 1);
+    let meetId = meetIdCell.getValue().trim();
+    if (meetId === "") {
+      // Meeting is not set
+      noMeetingRows.push(rowNumber);
+      continue;
+    }
+
+    let event = getEvent(setupResult.calendar, meetId);
+    if (!event) {
+      invalidMeetings.push(rowNumber);
+      continue;
+    }
+
+    let eventResource = Calendar.Events.get(calendarId, meetId);
+    if (!eventResource) {
+      console.log("Event could not be retrieved - " + calendarId + ", " + meetId);
+      invalidMeetings.push(rowNumber);
+      continue;
+    }
+
+    let eventAttachments = eventResource.attachments;
+    
+    let recordingLinkCell = setupResult.activeSheet.getRange(rowNumber, RECORDING_LINK_INDEX + 1);
+    let chatLinkCell = setupResult.activeSheet.getRange(rowNumber, CHAT_LINK_INDEX + 1);
+
+    if (!eventAttachments) {
+      recordingLinkCell.setValue(NOT_FOUND);
+      chatLinkCell.setValue(NOT_FOUND);
+      noRecordingLinks.push(rowNumber);
+      noChatLinks.push(rowNumber);
+      continue;
+    } else {
```

Comment:
No need of `else` here. The `continue` statement in the `if` block will ensure that this code won't be executed.

Reply:
I have added `continue` later and forgot to remove `else` part
Now, Updated the Requirement